modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -34,48 +35,38 @@ public static partial class Mathf
         /// </summary>
         public const real_t NaN = real_t.NaN;
 
-        private const real_t Deg2RadConst = (real_t) 0.0174532925199432957692369077M; // 0.0174532924f and 0.0174532925199433
-        private const real_t Rad2DegConst = (real_t) 57.295779513082320876798154814M; // 57.29578f and 57.2957795130823
+        private const real_t
+            _deg2RadConst = (real_t)0.0174532925199432957692369077M; // 0.0174532924f and 0.0174532925199433
```

Comment:
If this is going to overflow onto 2 lines, better to just move the comment to its own line. In fact, we should likely do this with other instances of `; //` too.

Reply:
I don't think the comment is necessarily bad there, but the column where it decides to break definitely is. It should have been at `=`.